src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1425,6 +1426,39 @@ public Section getSection(int id) {
             tmpl("/help_center/sections/{sectionId}/translations.json").set("sectionId", sectionId),
             handleList(Translation.class, "translations"));
     }
+    
+    
+    /**
+     * OrganizationMemberships method retrieves the organization memberships of user
+     * @param id
+     * @return OrganizationMembership
+     */
+    public List<OrganizationMembership> getOrganizationMemberships(long id) {
```

Comment:
Can we use `PagedIterable` instead of `List`? The API does return a `next_page` entry in the JSON record, so it may only return a partial result for users who are associated with many organizations (e.g. consultants).